smoketests/tests/permissions.py

```diff
@@ -21,28 +19,23 @@ def test_call(self):
     def test_delete(self):
         """Ensure that you cannot delete a database that you do not own"""
 
-        self.new_identity()
-
         self.publish_module()
 
-        self.reset_config()
+        self.new_identity()
```

Comment:
the goal of the `reset_config` was to be using a different identity than the `publish`. This accomplishes that directly.